karabo_data/tests/make_examples.py

```diff
@@ -255,12 +256,14 @@ def make_spb_run(dir_path, raw=True):
                 [
                     XGM('SA1_XTD2_XGM/DOOCS/MAIN'),
                     XGM('SPB_XTD9_XGM/DOOCS/MAIN'),
+                    BaslerCamera('SPB_IRU_SIDEMIC_CAM'),
```

Comment:
the naming convention is not respected in the camera you saw, but I think it would be better if you make up a name that respect it here. e.g. `SPB_IRU_CAM/CAM/SIDEMIC`

Reply:
@tmichela you have a reference dataset?